datajoint/blob.py

```diff
@@ -210,6 +213,11 @@ def __str__(self):
 
 
 def pack(obj, compress=True):
+    if bypass_serialization is True:
+        assert(any([obj[:len(pattern)] == pattern
```

Comment:
```suggestion
        assert obj.startswith(tuple(decode_loookup))
```
`assert` is not a function. Any can take an iterator and quit as soon as condition is met without forming a list. `str.startswith` can take a tuple of strings.